src/app/components/cards/Comment.jsx

```diff
@@ -300,11 +300,11 @@ class CommentImpl extends React.Component {
         // hide images if author is in blacklist
         const hideImages = ImageUserBlockList.includes(author);
 
-        const showDeleteOption = username === author && allowDelete;
+        const _isPaidout = comment.cashout_time === '1969-12-31T23:59:59'; // TODO: audit after HF19. #1259
```

Comment:
The comment here can probably be removed. We're post HF20 and the time hasn't been changed. 

Were it within scope, that string `'1969-12-31T23:59:59'` should be hoisted as a const and kept somewhere with good documentation.